src/memote/suite/cli/runner.py

```diff
@@ -324,7 +314,11 @@ def new(directory, replay):
     if directory is None:
         directory = os.getcwd()
     cookiecutter(
-        "gh:opencobra/cookiecutter-memote", output_dir=directory, replay=replay
+        # TODO: update cookiecutter at opencobra
+        "gh:carrascomj/cookiecutter-memote",
```

Comment:
How do we handle this? Do we first get your PR on the original cookiecutter repo merged? I would not like to merge this link, if possible.

Reply:
I think it is better if I resolve your comments in the sister cookiecutter PR (https://github.com/opencobra/cookiecutter-memote/pull/21, for reference). When both are ready, we can merge first the cookiecutter one, then commit the change here and merge this one afterwards.

Regarding the device flow, I am tempted to try to merge this one and then change it in a future PR but both ways are breaking user-facing changes, which is far from desirable. I'll study the device flow over the weekend and get back to this.

Reply:
I have taken a look at what we need for the device flow is

1. We need to [create a Github App](https://docs.github.com/en/enterprise-server@3.6/apps/creating-github-apps/registering-a-github-app/registering-a-github-app), public, with "Read & write" access to user repositories, installed for "Any account". I don't have permissions for doing that inside opencobra.
2. `memote online` requires the `client_id` from the Github App.
3. With the client ID, a POST request has to be sent to Github, the user will receive a code in the terminal which will have to be pasted in the browser.
4. At this point, memote receives the Github token and can be used as implemented in the PR ([gist sketch](https://gist.github.com/nat/d5cb7f356cdb677aa6cf3eecdd20a74c))

This is easier for the first-user experience but they will probably need to go through this process every time `memote online` is run. In the end, both ways can coexist in the implementation, so it may be OK to merge as is and implement the device workflow on top of it (would not be a breaking change w.r.t. this implementation after all). Let me know what you think is best.

Reply:
Great, thank you for looking into it. Then let's merge as is and we can add the device flow on top afterwards.